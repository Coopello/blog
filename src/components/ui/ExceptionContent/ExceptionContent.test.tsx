import renderer from "react-test-renderer";

import { ExceptionContent } from "./";

describe("ui/ExceptionContent", () => {
  it("Snap Shot", () => {
    const component = renderer.create(
      <ExceptionContent
        message={"テスト"}
        buttonComponent={<button>テスト</button>}
      />
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
