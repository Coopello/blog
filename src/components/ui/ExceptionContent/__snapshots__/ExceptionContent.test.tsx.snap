// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ui/ExceptionContent Snap Shot 1`] = `
<div
  className="grid min-h-[calc(100vh_-_60px)] w-full place-content-center gap-12 pb-20 sm:min-h-[calc(100vh_-_112px)]"
>
  <div>
    <img
      alt="エラーアイコン"
      className="mb-6"
      data-nimg="1"
      decoding="async"
      height={256}
      loading="lazy"
      onError={[Function]}
      onLoad={[Function]}
      src="/static/error_icon.svg"
      style={
        {
          "color": "transparent",
        }
      }
      width={256}
    />
    <h2
      className="text-2xl text-custom-primary"
    >
      テスト
    </h2>
  </div>
  <button>
    テスト
  </button>
</div>
`;
